Blockzilla/URLBar.swift

```diff
@@ -1010,19 +1010,19 @@ class TrackingProtectionBadge: UIView {
         trackingProtectionOn.setContentHuggingPriority(.required, for: .horizontal)
         trackingProtectionOn.snp.makeConstraints { make in
             make.centerX.centerY.equalToSuperview()
-            make.width.equalTo(UIConstants.layout.urlBarButtonImageSize)
+            make.width.equalTo(UIConstants.layout.shieldIconSize)
```

Comment:
I think we should rename this variable because it's used for multiple icons. 'urlButtonSize' should do.